deployment/CheckRequirements.ps1

```diff
@@ -37,15 +38,20 @@ if ($IncludeDev.IsPresent) {
 }
 
 # Powershell version
-$PowershellVersion = (Get-Host | Select-Object Version).Version
-if ($PowershellVersion -gt $PowershellMaxVersion) {
-    Add-LogMessage -Level Fatal "Please downgrade Powershell to a minimum version of $PowershellMinVersion and a maximum of $PowershellMaxVersion (currently using $PowershellVersion)!"
-} elseif ($PowershellVersion -lt $PowershellMinVersion) {
-    Add-LogMessage -Level Fatal "Please upgrade Powershell to a minimum version of $PowershellMinVersion and a maximum of $PowershellMaxVersion (currently using $PowershellVersion)!"
+$PowershellVersion = $PSVersionTable.PSVersion
+if ($PowershellVersion -lt $PowershellMinVersion) {
+    Add-LogMessage -Level Fatal "Please upgrade Powershell to a minimum version of $PowershellMinVersion (currently using $PowershellVersion; currently supported version is $PowershellSupportedVersion)!"
```

Comment:
As @jemrobinson said, it is probably better to remove the minimum version all together.

I think it is a bit misleading because we don't actually test or make any guarantees with versions between `$PowershellMinVersion` and `$PowershellVersion`.

Reply:
Okay, so remove all checks for Powershell version and just specify it in the documents, or leave in a soft check for the suggested version?

Reply:
I would leave the warning in. It will be more visible to users.